src/components/ConnectorName.tsx

```diff
@@ -15,6 +15,14 @@ function ConnectorName({ connector, iconPath, iconSize }: Props) {
         typeof connector === 'string' ? connector : getConnectorName(connector)
     );
 
+    const connectorIcon = useConstant(() =>
+        typeof iconPath === 'string'
```

Comment:
Classic `if`, `else if`, and `else` clauses may make this chain of conditionals easier to read. Mildly amused that Prettier does not indent the second evaluation.

Reply:
I agree... I only did this insanity to just keep the auto return there.